index.js

```diff
@@ -107,10 +105,20 @@ class NetworkResource extends Nanoresource {
     this.discovery.lookup(key, cb)
   }
 
+  _bindWrap (cb) {
```

Comment:
you should call this.options.bind() in ._open instead, just before it calls the cb

Reply:
:+1: